### DISCUSSION TITLE
Management PipeCD v0.44.0 features

### DISCUSSION THREAD

**Discussion by kentakozuka** [pipe-cd/pipecd#4367] [General] [answered]
About PipeCD release cycle: [Release docs](https://github.com/pipe-cd/pipecd/blob/master/RELEASES.md#release-cycle)
v0.44.0 project board: [v0.44.0](https://github.com/orgs/pipe-cd/projects/2/views/1)

Any other ideas/requests are more than welcome. Feel free to add as comments under this discussion 🙏

**Comment by khanhtc1202** [comment, marked as answer]
Released this version today 🙌
https://github.com/pipe-cd/pipecd/releases/tag/v0.44.0
